locales/en-US/experiments.ftl

```diff
@@ -111,18 +111,19 @@ nomore404sDetails1Copy = Brought to you by our friends at the Internet Archive.
 nomore404sContributors1Title = Developer, Wayback Machine, The Internet Archive
 nomore404sContributors2Title = Director, Wayback Machine, The Internet Archive
 nomore404sContributors3Title = Senior Crawl Engineer, The Internet Archive
-notesDescription = A simple notepad built into Firefox. Ever need to jot something down while you browse the web? This is the experiment for you.
-notesIntroductionMultinote = Notes adds a simple notepad to your browser’s sidebar. Sign in to your Firefox Account to sync all of your notes with end-to-end encryption.
+notesDescriptionMobile = Take notes as you browse the web, or keep in sync with our our new Android app.
```

Comment:
Is 'or' correct here?

Reply:
Nit: this says “... with our our new Android...” here (double “our”)

Reply:
D'oh, too focused on the 'or'